ui/src/components/BuildConfig.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import { Row, Col } from 'patternfly-react';
+import ComponentLabel from './ComponentLabel';
+
+import '../style/BuildConfig.css';
+
+class BuildConfig extends Component {
+  render() {
+    return (
+      <div className="build-config">
+        <ComponentLabel>
```

Comment:
https://github.com/aerogear/mobile-client-service/pull/19#discussion_r205737032 Same. I don't mind which one we use just so we remember to sync them up